homeassistant/components/ampio/air_quality.py

```diff
@@ -43,50 +54,52 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class AmpioSmogQuality(AirQualityEntity):
     """Implementation of an Ampio Smog air quality entity."""
 
-    def __init__(self, api, station_id, name):
+    def __init__(
+        self, api: AmpioSmogMapData, station_id: str, name: str | None
+    ) -> None:
         """Initialize the air quality entity."""
         self._ampio = api
         self._station_id = station_id
         self._name = name or api.api.name
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the air quality entity."""
         return self._name
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return unique_name."""
         return f"ampio_smog_{self._station_id}"
 
     @property
-    def particulate_matter_2_5(self):
+    def particulate_matter_2_5(self) -> str | None:
         """Return the particulate matter 2.5 level."""
-        return self._ampio.api.pm2_5
+        return self._ampio.api.pm2_5  # type: ignore[no-any-return]
 
     @property
-    def particulate_matter_10(self):
+    def particulate_matter_10(self) -> str | None:
```

Comment:
Same here.

Reply:
as per baseclass `AirQualityEntity` `particulate_matter_2_5` seems to be requiered, but other `particulate_matter_xxx` only optional (_can return `None`_ )